github/ci/prow-deploy/kustom/base/configs/current/labels/labels.yaml

```diff
@@ -96,6 +96,12 @@ default:
       target: both
       addedBy: anyone
       prowPlugin: label
+    - name: kind/flake
```

Comment:
The naming does not differentiate between handling tests that are flake and production code that is flaky.
Is this intentional?

Reply:
BTW production code that is flaky is a bug to me, no?

Reply:
Yes, but it is also a flake.
I mean, it may or may not be detected by tests. We had such cases.

Reply:
To fix the above scenario the test can be left unchanged, but the production code needs to get fixed.